scenario.py

```diff
@@ -130,7 +130,7 @@ def __init__(self, params, experiment_path, scenario_id=1, n_repeat=1, is_dry_ru
         # ---------------------------------------------------
 
         self.mpl = None
-        self.evaluation_partner_number = None
+        self.evaluation_partner_numbers = None
         self.sequential_weighting_ponderation = None
```

Comment:
It is quite difficult to understand what this parameter is for. Could you clarify @MrDaiki and add a comment about it?